src/workerd/io/worker.c++

```diff
@@ -1213,25 +1213,14 @@ Worker::Isolate::Isolate(kj::Own<Api> apiParam,
               [reactions = v8::Global<v8::Data>(isolate, reactions),
                   argument = jsg::JsRef(js, jsg::JsValue(argument)),
                   callback = kj::mv(callback)](jsg::Lock& js) mutable {
-            try {
-              callback(js.v8Isolate, reactions.Get(js.v8Isolate), argument.getHandle(js));
-            } catch (jsg::JsExceptionThrown&) {
-              // Exceptions here are generally unexpected but possible. Let's just
-              // return control back to V8 which should handle the case.
-            } catch (...) {
-              auto ex = kj::getCaughtExceptionAsKj();
-              KJ_LOG(ERROR, "Scheduling deferred promise continuations failed unexpectedly", ex);
-              jsg::throwInternalError(js.v8Isolate, kj::mv(ex));
-            }
+            callback(js.v8Isolate, reactions.Get(js.v8Isolate), argument.getHandle(js));
```

Comment:
Oh it occurs to me, this is a callback _to_ v8, so it can't possibly throw anyway, right? So more reason why it was right to remove the try/catch.

Though it raises a different question: What if the callback failed, e.g. due to termination? That doesn't seem to be signaled at all. Does the callback actually need to return a bool or something?

Reply:
I looked at this but the only thing the callback does is call the function to schedule the microtasks and that always returns `undefined`. It does not appear to signal any failure conditions and does not appear to return an empty handle to signal any kind of failure.

Reply:
Fair enough.